### DISCUSSION TITLE
Support React 19

### DISCUSSION THREAD

**Discussion by jkone27** [Zaid-Ajaj/Feliz#638] [Ideas]
react 19 seems to add quite many cool features, 

https://vercel.com/blog/whats-new-in-react-19 

would be great to have Feliz / Fable work with react-19

**Comment by Freymaurer** [comment]
It is supported now with Feliz v3!

**Reply by laurentpayot** [reply]
Also working with Preact v11.0.0-beta.0! :tada:

**Reply by laurentpayot** [reply]
Thank you so much @Freymaurer for all the work done!
